package.json

```
@@ -19,7 +19,7 @@
         ],
         "nrfutil": {
             "device": [
-                "2.5.0"
+                "2.6.4"
```

```suggestion
                "2.7.3"
```
The new version of nrfutil-device fixes a bug preventing board configurator from reading and writing config.